awbw-notes.user.js

```diff
@@ -15,26 +15,74 @@
     var text = localStorage.getItem(key) || "";
 
     var textBox = document.createElement("textarea");
-    textBox.setAttribute("class", "playerNotes");
-    textBox.setAttribute("style", "width:200px;height:100px;");
+    textBox.setAttribute("class", "notesClosable");
+    textBox.setAttribute("style", "position:inline;width:200px;height:100px;");
     textBox.value = text;
 
+    var closeButton = document.createElement("div");
+    closeButton.innerHTML = "&#9660;"
+    closeButton.setAttribute("id", "notesClose");
+
     var divBox = document.createElement("div");
+    divBox.innerHTML = "<header class='notesClosable'>Notes:</header>";
     divBox.setAttribute("id", "playerNotesArea");
-    divBox.setAttribute("style", "position:fixed;bottom:0px;right:0px;");
-    divBox.appendChild(textBox)
+    divBox.prepend(closeButton);
+    divBox.appendChild(textBox);
 
     var mapControls = document.querySelector("#map-controls-container");
     insertAfter(divBox, mapControls);
 
     textBox.addEventListener("change", function(event) {
         localStorage.setItem(key, textBox.value);
     });
-    
+
     textBox.addEventListener("keydown", function(event) {
         event.stopPropagation();
     });
 
+    var isOpen = true;
+    closeButton.addEventListener("click", function(event) {
+        if(isOpen) {
+            var closableThings = document.getElementsByClassName("notesClosable");
+            for (var i = 0; i < closableThings.length; i++) {
+                closableThings[i].style.display = "none";
+            }
+            closeButton.innerHTML = "&#9650;"
+            isOpen = false;
+        } else {
+            var closableThings = document.getElementsByClassName("notesClosable");
+            for (var i = 0; i < closableThings.length; i++) {
+                closableThings[i].style.display = "block";
+            }
+            closeButton.innerHTML = "&#9660;"
+            isOpen = true;
+        }
+     });
+
+    GM_addStyle(`
+        .playerNotes {
+            width: 90%
+        }
```

Comment:
I think no element with this class exists anymore?

Reply:
Yep, totally correct. Lemme remove it.